examples/shop-page/Components/ItemList.lua

```diff
@@ -0,0 +1,28 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+local Components = script.Parent
+
+local Roact = require(ReplicatedStorage.Roact)
+
+local Item = require(Components:WaitForChild("Item"))
```

Comment:
Why do we need WaitForChild here? Is it unsafe for children of a folder to assume its siblings are available?

Reply:
I don't think there is any guarantee that all the siblings are replicated. From my experience it's best to use WaitForChild since you're sure everything is there when you need it.

Maybe @LPGhatguy can validate this!